index.html

```diff
@@ -5,44 +5,129 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="css/style.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
 </head>
 <body>
    <header>
+       <div class="upper_navbar">
+           <ul>
+            <li><a href="#"><img src="images/linkedinlogo.png" alt="Linked in nav item"></a></li>
+            <li><a href="#"><img src="images/twitterlogo.png" alt="Linked in nav item"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>          
+         </ul>
+       </div>
        <nav>
-           <button type="button"><img src="images/hamburger.png" alt="Hamburger button"></button>
-       </nav>
+           <button type="button" id="hamburger-btn"><img src="images/hamburger.png" alt="Hamburger button"></button>
```

Comment:
- [ ] [OPTIONAL] The color in the mobile menu is strong kindly consider adding some opacity to the background color or changing it to a less strong color

![Screenshot_20221028_222151](https://user-images.githubusercontent.com/25227022/198736262-e1ca7228-8653-43f9-808d-c0135375a146.png)